frontend/src/components/Auth/SignupForm.js

```diff
@@ -63,17 +65,17 @@ const CreateUserForm = ({ url }) => {
               onChange={(e) => setFormData({ ...formData, password: e.target.value })}
             />
             {error.password ? (
-              <div className="ui pointing basic label" style={errorStyle}>
+              <div className={formClasses.fieldError}>
                 {error.password}
               </div>
             ) : null}
           </div>
 
           {error.non_field_errors ? (
-            <div className="ui negative message" style={errorStyle}>
-              {error.non_field_errors}
-            </div>
-          ) : null}
+          <center><div className={formClasses.error}>
+            { error.non_field_errors }
+          </div><br></br><br></br></center>
```

Comment:
You can use ie. `formClasses.lastError` and define within property `margin-bottom: 50px`